docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/the-git-based-development-process-for-mainframe-development.md

```diff
@@ -161,11 +161,11 @@ At a high level, the development team works through the following tasks:
 
 1.  New work items are managed in the backlog. The team decides which work items will be implemented in the next iteration. Each application team can decide about the duration of the iteration (which can also be seen as the development cycle). In the above diagram, three work items were selected to be implemented for the next iteration. The development team is responsible for coordinating if features are demanding to be implemented in a specific order.
 
-2.  For each work item, a feature branch is created according to pre-defined naming conventions, allowing the assigned developers to have a copy of the codebase on which they can work in isolation from other concurrent development activities.
+2.  For each work item, a feature branch is created according to pre-defined naming conventions, allowing the assigned developers to work in isolation from other concurrent development activities.
 
 ![](images/media/image5.png)
 
-3.  To start making the necessary modifications for their development task, developers create a copy of the Git repository on their local workstations through the clone operation of Git. To synchronize their local Git repository, with the central Git repository they fetch or pull updates into their local clone of the repository. With this process developers make the feature branch available in their local clone.[^6]
+3.  To start making the necessary modifications for their development task, developers create a copy of the Git repository on their local workstations through the clone operation of Git. They checkout or switch to the feature branch. To synchronize their local Git repository, with the central Git repository they fetch or pull updates into their local clone of the repository. With this process developers make the feature branch available in their local clone.[^6]
```

Comment:
> They checkout or switch to...

Revise to:
"They check out or switch to..."

Reply:
This is more a rewording of the step in general (rather than just the proposal in the PR), but how about:

> 3. To start making the necessary modifications for their development task, developers create a copy of the Git repository on their local workstations through the clone operation of Git. They then  switch to the feature branch.
>    - If the developer already has a local copy of the Git repository, they can simply synchronize their local Git repository with the central Git repository by fetching or pulling updates into their local clone of the repository. This makes the feature branch available to switch to in their local clone.

Reply:
slight simplification:

> This makes the feature branch available ~to switch to~ in their local clone.

Just to not labour the point about switching repeatedly.